graylog2-web-interface/src/views/components/export/ExportModal.test.tsx

```diff
@@ -204,28 +203,65 @@ describe('ExportModal', () => {
     );
   });
 
+  it('initial fields should be  in the same direction as in widgetConfig where are more than 8 fields', async () => {
```

Comment:
Just a suggestion for a little improvement:
`initial fields should keep order when there are more than 8 fields in widget config`